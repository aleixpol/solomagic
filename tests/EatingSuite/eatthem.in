\ref something0
\mt somethingsomething andri somethingsomething

\ref something1
\mt somethingsomething andrian somethingsomething

