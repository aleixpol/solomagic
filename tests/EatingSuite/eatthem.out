\ref something0
\mt somethingsomething andri somethingsomething
\ma somethingsomething angri somethingsomething

\ref something1
\mt somethingsomething andre somethingsomething
\ma somethingsomething andre somethingsomething
